<!DOCTYPE html><html><head><title></title><meta http-equiv="Content-type" content="text/html;charset=UTF-8" /><meta name="viewport" content="width=device-width, initial-scale=1"><meta name="description" content="" /><meta name="generator" content="EverWeb 2.9.1 (2184)" /><meta name="buildDate" content="Wednesday, February 27, 2019" /><link rel="canonical" href="http://biophys.umontreal.ca/nrg/" /><meta property="og:url" content="http://biophys.umontreal.ca/nrg/" /><meta property="og:type" content="website" /> <link rel="stylesheet" type="text/css" href="ew_css/textstyles.css?3634140328" /><link rel="stylesheet" type="text/css" href="ew_css/responsive.css?3634140328" /><!--[if lt IE 9]>
<script src="//ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"></script>
<!--<![endif]-->
<!--[if (gte IE 9) | (!IE)]><!-->
<script src="//ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
<!--<![endif]-->
<script type="text/javascript">
if (typeof jQuery == 'undefined')
{
<!--[if lt IE 9]>
 document.write(unescape("%3Cscript src='ew_js/jquery-1.9.1.min' type='text/javascript'%3E%3C/script%3E"));
<!--<![endif]-->
<!--[if (gte IE 9) | (!IE)]><!-->
 document.write(unescape("%3Cscript src='ew_js/jquery.min.js' type='text/javascript'%3E%3C/script%3E"));
<!--<![endif]-->
}
</script><script type="text/javascript" src="ew_js/imageCode.js"></script><script type="text/javascript" src="ew_js/viewchecker.js"></script><link rel="stylesheet" type="text/css" href="ew_css/animate.css" /><link rel="stylesheet" type="text/css" href="//fonts.googleapis.com/css?family=Work+Sans:400,700,300"><style>
	div#text_box_7 .tb_button { font-family:'WorkSans-Regular','Work Sans','Helvetica';font-size:16px;color:#D5D5D5;text-decoration:none; }
	
</style><style>div#text_box_7 a {color: rgb(4,50,255);text-decoration: underline;}div#text_box_7 a:hover {color: rgb(255,38,0);text-decoration: underline;}div#text_box_7 a:visited {color: rgb(148,32,146);text-decoration: underline;}</style><script type="text/javascript">jQuery(document).ready(function(){jQuery(".animated").addClass("animhidden").viewportChecker({callbackFunction:function(a){var b=a.data("fadeopt");a.hasClass(b)||a.addClass(b)},classToAdd:"animvisible",classToRemove:"animhidden",offset:100})});</script><style>
		div#ewfx-flex-container_9 *  {
			box-sizing: border-box;
			
			
		}
		
		div#ewfx-flex-container_9  {
			width:100%;
			display: flex;
			flex-wrap: wrap;
			flex-direction: row;
			justify-content: center; 
			align-items: center;
		}
		
		div#ewfx-flex-container_9 .ewfx-row {
			flex-shrink: 2;
			flex-grow: 2;
		
		}
		
		div#ewfx-flex-container_9 .ewfx-row .ewfx-videoWrapper {
			position: relative;
			padding-bottom: 56.25%; /* 16:9 */
			padding-top: 25px;
			height: 0;
		}
		div#ewfx-flex-container_9 .ewfx-row .ewfx-videoWrapper iframe {
			position: absolute;
			top: 0;
			left: 0;
			width: 100%;
			height: 100%;
		}

	</style><style>div#ewfx-flex-container_9 a {color: rgb(4,50,255);text-decoration: underline;}div#ewfx-flex-container_9 a:hover {color: rgb(255,38,0);text-decoration: underline;}div#ewfx-flex-container_9 a:visited {color: rgb(148,32,146);text-decoration: underline;}</style><style type="text/css">a img {border:0px;}div.content {min-height:1000px;}footer  {min-height:100px;}body {background: url('images/home/browser/1stp.png') no-repeat center center fixed;-webkit-background-size: cover;-moz-background-size: cover;-o-background-size: cover;background-size: cover;background-color: #FEFFFF;margin: 0px 0 0 0;}div.container {width: 100%;}@media all and (max-device-width: 768px),(max-device-width: 568px),(max-device-width: 480px),(max-device-width: 667px),(max-device-width: 736px),(max-device-width: 360px)   {body {background: url('images/home/browser/1stp-768px.png') no-repeat center center fixed;-webkit-background-size: cover;-moz-background-size: cover;-o-background-size: cover;background-size: cover;background-color: #FEFFFF;}}#flexbox_back_0{background-color:rgba(0,0,0,0.8);width:100%;height:100%;padding-top:25px;padding-bottom:25px;min-height:130px;}#flexbox_0{display:-webkit-flex;;display:-moz-box;display:-ms-flexbox;display:flex;-webkit-flex-wrap:wrap;flex-wrap:wrap;z-index:0;justify-content:center;align-items:center;}#flexbox_0 > div{padding:15px;}#flexbox_back_1{background-color:rgba(145,145,145,0.9);width:100%;height:100%;padding-top:75px;padding-bottom:75px;min-height:314px;}#flexbox_1{display:-webkit-flex;;display:-moz-box;display:-ms-flexbox;display:flex;-webkit-flex-wrap:wrap;flex-wrap:wrap;z-index:1;justify-content:center;align-items:flex-start;}#flexbox_1 > div{padding:0px;}#flexbox_back_2{background-color:rgba(254,255,255,1);width:100%;height:100%;padding-top:0px;padding-bottom:0px;}#flexbox_2{display:-webkit-flex;;display:-moz-box;display:-ms-flexbox;display:flex;-webkit-flex-wrap:wrap;flex-wrap:wrap;z-index:2;justify-content:space-around;align-items:center;}#flexbox_2 > div{padding:0px;}#flexbox_back_3{background-color:rgba(44,63,84,0.5);width:100%;height:100%;padding-top:200px;padding-bottom:200px;}#flexbox_3{display:-webkit-flex;;display:-moz-box;display:-ms-flexbox;display:flex;-webkit-flex-wrap:wrap;flex-wrap:wrap;z-index:3;justify-content:center;align-items:center;}#flexbox_3 > div{padding:0px;}#flexbox_back_4{background-color:rgba(254,255,255,1);width:100%;height:100%;padding-top:20px;padding-bottom:20px;min-height:112px;}#flexbox_4{display:-webkit-flex;;display:-moz-box;display:-ms-flexbox;display:flex;-webkit-flex-wrap:wrap;flex-wrap:wrap;z-index:4;justify-content:space-around;align-items:center;}#flexbox_4 > div{padding:0px;}#navmenu_menu0 {clear:both;float:left;margin:0;padding:0;width:100%;font-family:'WorkSans-Light','Work Sans','Helvetica';font-size:14px;z-index:1000;}#navmenu_menu0 ul {margin:0;padding:0;list-style:none;position:relative;right:0px;float:right;}#navmenu_menu0 ul li {margin:0 10px 0 0;padding:0;float:left;position:relative;left:0px;min-width: 15px;}#navmenu_menu0 ul li a {display:block;margin:0;padding:5px;height: 27px;line-height: 27px;text-decoration:none;color:#A9A9A8;font-weight: 300;text-align: left;}#navmenu_menu0 ul li a:visited {text-decoration:none;color:#A9A9A8;}#navmenu_menu0 ul li a:hover {text-decoration:none;color:#FE7D77;}#navmenu_menu0 ul li:hover a,#navmenu_menu0 ul li.hover a {text-decoration:none;color:#FE7D77;}#navmenu_menu0 ul ul {z-index:11006;display:none;position:absolute;left:0;float:left;right:auto;padding-top:2px;}#navmenu_menu0 ul ul li {left:auto;margin:0;clear:left;float:left;width:100%;}#navmenu_menu0 ul ul li a,#navmenu_menu0 ul li.active li a,#navmenu_menu0 ul li:hover ul li a,#navmenu_menu0 ul li.hover ul li a {background:#CBCBCB;text-decoration:none;color:#A9A9A8;padding-top:0px;padding-bottom:0px;border-bottom:1px solid #000000;float:left;width:5px;width:100%;text-align: left;white-space:nowrap;}#navmenu_menu0 ul ul li a:hover,#navmenu_menu0 ul li.active ul li a:hover,#navmenu_menu0 ul li:hover ul li a:hover,#navmenu_menu0 ul li.hover ul li a:hover {background:#CBCBCB;text-decoration:none;color:#FE7D77;float:left;}#navmenu_menu0 ul li.last ul {left:auto; right:0;}#navmenu_menu0 ul li.last ul li {float:right;position:relative;right:10px;}#navmenu_menu0 ul li:hover ul,#navmenu_menu0 ul li.hover ul {display:block;}</style></head><body><div class="container"><div class="content" data-minheight="1000"><div class="" id="flexbox_back_4"><div id="flexbox_4"><div><div class="shape_5" style="width:355px;min-height:112px;"><div style="padding: 42px 2.16px 0px 2.16px; "><p style="line-height:28px;text-align:center;margin-top:0px;margin-bottom:0px;" class="Style1">N<span class="Style8">ajmanovich</span><span class="Style9"> R</span><span class="Style8">esearch</span><span class="Style9"> G</span><span class="Style8">roup</span></p></div></div></div><div><div class="ewnavmenu" id="navmenu_menu0" style="left:858px;top:57px;height:37px;width:360px;z-index:10006;"><ul id="navigation_menu0"><li><a class="main item0" href="index.html" data-linkuuid="597C71A1D6894442B65047670AB54CF4">Home</a></li><li><a class="main item1" href="resources.html" data-linkuuid="597C71A1D6894442B65047670AB54CF4">Resources</a></li><li  style='margin-right:0px;' class="last"><a class="main item2" href="publications.html" data-linkuuid="597C71A1D6894442B65047670AB54CF4">Publications</a></li></ul></div></div></div></div><div class="" id="flexbox_back_3"><div id="flexbox_3"><div><section style="width:100%;animation-duration:1.5s;-webkit-animation-duration:1.5s;animation-delay:0.0s;-webkit-animation-delay:0.0s;animation-iteration-count:1;-webkit-animation-iteration-count:1;" data-fadeopt="fadeInUp" class="  animated"><!--1385-->
	<div id="text_box_7" style="text-align:Center;min-width:300px"">
	<br />
	
	<span class="ql-font-helvetica" style="font-size: 12px;font-family:'Helvetica',sans-serif;"><p><strong style="font-size: 44px; color: rgb(255, 255, 255);">Computational Structural Pharmacology</strong></p><p><em style="font-size: 26px; color: rgb(255, 255, 255);">Molecular design from molecules to systems</em></p></span>

	
	<div style="display: flex;flex-direction:row;flex-wrap: wrap;justify-content: center;align-items:center;">
	<div style="flex-grow:2;flex-shrink:2;margin:15px;min-width:150px;max-width:250px;"><a target="_blank" style="text-decoration:none;" href="resources.html" data-linkuuid="597C71A1D6894442B65047670AB54CF4"><div class="tb_button" style='background-color: #FB0106;padding: 15px 10px 15px 10px;border-radius: 50px;text-decoration:none;text-align:center;color: #FFFFFF;'>Resources</div></a></div><div style="flex-grow:2;flex-shrink:2;margin:15px;min-width:150px;max-width:250px;"><a target="_blank" style="text-decoration:none;" href="publications.html" data-linkuuid="597C71A1D6894442B65047670AB54CF4"><div class="tb_button" style='background-color: #FF0000;padding: 15px 10px 15px 10px;border-radius: 50px;text-decoration:none;text-align:center;color: #FFFFFF;' onmouseover="this.style.background='#FB0106';this.style.color='#FFFFFF';this.style.border='0px solid #000000';" onmouseout="this.style.background='#FF0000';this.style.color='#FFFFFF';this.style.border='0px solid #000000';">Publications</div></a></div>
	</div>
	
	</div></section></div></div></div><div class="" id="flexbox_back_2"><div id="flexbox_2"><div><section style="width:100%;" class=""><!--1385-->
		
		<div id="ewfx-flex-container_9">
		<div class="ewfx-row" style="flex-basis: 90%;max-width: 90%;min-width: 40%;margin: 50px;align-self: center;">
<div class="ewfb-container-1"><div class="ewfb-overlay"><div class="ewfb-image-desc"><span class="ql-font-helvetica" style="font-size: 12px;font-family:'Helvetica',sans-serif;"><p><strong style="font-size: 32px; color: rgb(25, 25, 25);">Our Research</strong></p><p class="ql-align-justify" style="text-align:justify"><span style="color: rgb(136, 136, 136); background-color: rgb(255, 255, 255); font-size: 14px;">
	Our research traverses from genomes to small molecules integrating systems, structural and computational pharmacology as well as chemo- and bioinformatics. Our work is divided into four interconnected but independent axes within which we combine the development and use of innovative computational methods with experimentally validation. Namely: 1. The reconstruction and simulation of metabolic networks; 2. The detection of binding-site structural similarities; 3. Simulation of dynamic aspects of protein function; and 4. The development of docking algorithms.</span></p>

	<p class="ql-align-justify" style="text-align:justify"><span style="background-color: rgb(255, 255, 255); color: rgb(68, 68, 68); font-size: 14px;">AXIS 1: SYSTEMS PHARMACOLOGY</span></p>
	<p class="ql-align-justify" style="text-align:justify"><span class="ql-font-helvetica" style="font-size: 14px; background-color: rgb(255, 255, 255); color: rgb(136, 136, 136);font-family:'Helvetica',sans-serif;">
	We construct genome-wide metabolic networks and use them to simulate metabolism at a system-wide level. We developed the first curated metabolic network for C. difficile and used it to predict essential therapeutic targets (with 93% accuracy with experiments) for the development of new antibiotics. We integrated the metabolic simulations with structural biology to detect potential off-targets in humans and detect potential existing inhibitors. In certain cases, these inhibitors are approved drugs for other conditions that may be repositioned to fight C. difficile. We have also developed a metabolic network of human adipocytes to detect potentialtherapeutic targets that may remodel the metabolism of adipocyte. Our goal is to detect targets that may redirect pre-diabetic cells from hyperplasia to hypertrophy, a less damaging condition to cope with the excess circulating fatty acids and prevent the onset of Type 2 diabetes. Future plans are to create robust cellular models for the other human tissues with the addition of signalling and regulatory networks. These models will include structural and protein dynamics information for all proteins that can be accurately modelled. Variability between individuals in off-target protein expression levels could in principle be a source of idiosyncratic drug responses. Lastly, we are working on the application of the DEVS methodology in collaboration with Carleton University to perform genome scale discrete event simulations. This methodology will eventually lead to the integration of metabolic, signalling and gene-regulatory networks as well as other biological processes in a seamless modular manner to perform whole-cell simulations, making it a reality the long promised goals of systems biology.</span></p>

	<p class="ql-align-justify" style="text-align:justify"><span style="background-color: rgb(255, 255, 255); color: rgb(68, 68, 68); font-size: 14px;">AXIS 2: DETECTION OF MOLECULAR SIMILARITIES</span></p>
<p class="ql-align-justify" style="text-align:justify"><span class="ql-font-helvetica" style="background-color: rgb(255, 255, 255); color: rgb(136, 136, 136); font-size: 14px;font-family:'Helvetica',sans-serif;">
	We developed two algorithms, IsoCleft and IsoMIF, to detect binding-site molecular similarities to predict cross-reactivity targets, polypharmacology targets and understand protein function within and across protein families. These methods have a broad range of applications. IsoMIF in particular is based on the detection of common interactionpatterns of chemical probes within binding sites irrespective of the arrangement or nature of amino acids. IsoMIF makes it possible to predict cross-reactivity targets at the functional level in the absence of similarities at the sequence or structural levels. The future development of this area involves considering flexibility explicitly through the incorporation of normal modes (Axis 3) to treat flexibility more realistically and detect similarities more accurately. We also plan to detect similarities among protein-protein interaction interfaces and adapt IsoMIF to detect molecular interaction field similarities around small-molecules as an alternative to 3D-QSAR methods that require the superimposition of highly similar scaffolds. In the case of 3D-QSAR, the use of IsoMIF will permit to generate models without requiring any scaffold similarity, thus considerably extending the utilityof3D-QSAR in drug design and toxicity modelling.</span></p>
	
	<p class="ql-align-justify" style="text-align:justify"><span style="background-color: rgb(255, 255, 255); color: rgb(0, 0, 0); font-size: 14px;">AXIS 3: DYNAMICS OF MACROMOLECULES</span></p>
	<p class="ql-align-justify" style="text-align:justify"><span class="ql-font-helvetica" style="color: rgb(136, 136, 136); font-size: 14px; background-color: rgb(255, 255, 255);font-family:'Helvetica',sans-serif;">We developed ENCoM, the first coarse-grained Normal Mode Analysis method to account for the effect of mutations on dynamics and thermal stability. We can use ENCoM to model large conformational changes such as domain and loop movements as well as those in large proteins. These large-scale movements are inaccessible through either Molecular Dynamics simulations or NMR thus providing complementary information to these techniques. We demonstrated that ENCoM can differentiate mesophile proteins from their thermophile homologs and guide the selection of mutations from one to the other. We plan to apply ENCoM in protein engineering to increase protein stability while maintaining flexibility for proteins of industrial and medical interest, particularly biologics. The possibility to jointly predict stability and flexibility is unique to ENCoM. Given the recognized importance of maintaining flexibility in protein design, ENCoM may prove to be a useful tool in protein engineering. ENCoM can also be used to predict constitutively active or inactive mutations in G-protein Coupled Receptors (GPCRs) or classify different types of ligands based on their effect on the dynamics of distinct regions of the GPCR. We are currently developing ENCoM to study RNA dynamics.</span></p>
	
	<p class="ql-align-justify" style="text-align:justify"><span style="background-color: rgb(255, 255, 255); color: rgb(0, 0, 0); font-size: 14px;">AXIS 4: DOCKING SIMULATIONS</span></p>
	<p class="ql-align-justify" style="text-align:justify"><span class="ql-font-helvetica" style="color: rgb(127, 127, 127); font-size: 14px; background-color: rgb(255, 255, 255);font-family:'Helvetica',sans-serif;">We developed FlexAID, a small-molecule protein docking method that performs better than widely used methods such as FlexX and AutoDock Vina against non native-complex structures, i.e. structures that were not crystallised with the ligand that one wants to dock (a realistic scenario), and performs better than rDock when side-chain flexibility is critical for binding. FlexAID was recently used to identify potent selective nM compounds that inhibit Matriptase and Matriptase-2 and the detection of small molecules from ZINC through virtual screening that selectively kill C. difficile targeting the guaA riboswitch (an RNA target). We also created the NRGsuite PyMOL interface that allows users to easily perform docking simulations. Our plans involve the introduction of full protein backbone movements to FlexAID using ENCoM enabling us to simulate the full range of protein movements on the fly during docking simulations. We also plan to take advantage of the genetic algorithm population of solutions to account for entropy in an integral and statistical mechanics well-defined way to rank solutions. We plan to apply FlexAID in the search for allosteric small-molecule inhibitors of protein activity and protein-protein interactions.</span></p>
	
	<p class="ql-align-justify" style="text-align:justify"><br></p></span></div></div></div></div>
		</div></section></div></div></div></div><footer data-top='1653' data-height='644'><div class="" id="flexbox_back_1"><div id="flexbox_1"><div><div class="shape_8" style="width:428px;min-height:314px;"><div style="padding: 36px 36px 36px 36px; "><p style="line-height:25px;text-align:center;margin-top:0px;margin-bottom:11px;" class="Style2">Contact Infromation</p><p style="line-height:19px;margin-bottom:11px;margin-top:0px;" class="Style3">Department of Pharmacology & Physiology</p><p style="line-height:19px;margin-bottom:11px;margin-top:0px;" class="Style3">Faculty of Medicine</p><p style="line-height:19px;margin-bottom:11px;margin-top:0px;" class="Style3">Universit√© de Montreal</p><p style="line-height:19px;margin-bottom:11px;margin-top:0px;" class="Style3"><br /></p><p style="line-height:19px;margin-bottom:11px;margin-top:0px;" class="Style3">Room 3147 - Pavillion Paul-G.-Desmarais</p><p style="line-height:19px;margin-bottom:11px;margin-top:0px;" class="Style3">2960 Chemin de la Tour - H3T 1J4</p><p style="line-height:19px;margin-bottom:11px;margin-top:0px;" class="Style3">Montreal, QC - Canada</p></div></div></div></div></div><div class="" id="flexbox_back_0"><div id="flexbox_0"><div><div class="shape_10" style="width:212px;min-height:100px;"><img src="images/home/Logo_UdeM-blanc-RVB.png" height="100" width="212" /></div></div></div></div></footer></div></body></html>