<!DOCTYPE html><html><head><title></title><meta http-equiv="Content-type" content="text/html;charset=UTF-8" /><meta name="viewport" content="width=device-width, initial-scale=1"><meta name="description" content="" /><meta name="generator" content="EverWeb 2.9.1 (2184)" /><meta name="buildDate" content="Wednesday, February 27, 2019" /><meta property="og:url" content="http://biophys.umontreal.ca/nrg/publications.html" /><meta property="og:type" content="website" /> <link rel="stylesheet" type="text/css" href="ew_css/textstyles.css?3634138158" /><link rel="stylesheet" type="text/css" href="ew_css/responsive.css?3634138158" /><!--[if lt IE 9]>
<script src="//ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"></script>
<!--<![endif]-->
<!--[if (gte IE 9) | (!IE)]><!-->
<script src="//ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
<!--<![endif]-->
<script type="text/javascript">
if (typeof jQuery == 'undefined')
{
<!--[if lt IE 9]>
 document.write(unescape("%3Cscript src='ew_js/jquery-1.9.1.min' type='text/javascript'%3E%3C/script%3E"));
<!--<![endif]-->
<!--[if (gte IE 9) | (!IE)]><!-->
 document.write(unescape("%3Cscript src='ew_js/jquery.min.js' type='text/javascript'%3E%3C/script%3E"));
<!--<![endif]-->
}
</script><script type="text/javascript" src="ew_js/imageCode.js"></script><script type="text/javascript" src="ew_js/viewchecker.js"></script><link rel="stylesheet" type="text/css" href="ew_css/animate.css" /><link rel="stylesheet" type="text/css" href="//fonts.googleapis.com/css?family=Work+Sans:400,700,300"><style>
	div#text_box_7 .tb_button { font-family:'ArialMT','Arial';font-size:16px;color:#D5D5D5;text-decoration:none; }
	
</style><style>div#text_box_7 a {color: rgb(4,50,255);text-decoration: underline;}div#text_box_7 a:hover {color: rgb(255,38,0);text-decoration: underline;}div#text_box_7 a:visited {color: rgb(148,32,146);text-decoration: underline;}</style><script type="text/javascript">jQuery(document).ready(function(){jQuery(".animated").addClass("animhidden").viewportChecker({callbackFunction:function(a){var b=a.data("fadeopt");a.hasClass(b)||a.addClass(b)},classToAdd:"animvisible",classToRemove:"animhidden",offset:100})});</script><style type="text/css">a img {border:0px;}div.content {min-height:1000px;}footer  {min-height:100px;}body {background: url('images/publications/browser/1stp.png') no-repeat center center fixed;-webkit-background-size: cover;-moz-background-size: cover;-o-background-size: cover;background-size: cover;background-color: #FEFFFF;margin: 0px 0 0 0;}div.container {width: 100%;}@media all and (max-device-width: 768px),(max-device-width: 568px),(max-device-width: 480px),(max-device-width: 667px),(max-device-width: 736px),(max-device-width: 360px)   {body {background: url('images/publications/browser/1stp-768px.png') no-repeat center center fixed;-webkit-background-size: cover;-moz-background-size: cover;-o-background-size: cover;background-size: cover;background-color: #FEFFFF;}}#flexbox_back_0{background-color:rgba(0,0,0,0.8);width:100%;height:100%;padding-top:25px;padding-bottom:25px;min-height:130px;}#flexbox_0{display:-webkit-flex;;display:-moz-box;display:-ms-flexbox;display:flex;-webkit-flex-wrap:wrap;flex-wrap:wrap;z-index:0;justify-content:center;align-items:center;}#flexbox_0 > div{padding:15px;}#flexbox_back_1{background-color:rgba(145,145,145,0.9);width:100%;height:100%;padding-top:75px;padding-bottom:75px;min-height:314px;}#flexbox_1{display:-webkit-flex;;display:-moz-box;display:-ms-flexbox;display:flex;-webkit-flex-wrap:wrap;flex-wrap:wrap;z-index:1;justify-content:center;align-items:flex-start;}#flexbox_1 > div{padding:0px;}#flexbox_back_2{background-color:rgba(254,255,255,1);width:100%;height:100%;padding-top:0px;padding-bottom:0px;}#flexbox_2{display:-webkit-flex;;display:-moz-box;display:-ms-flexbox;display:flex;-webkit-flex-wrap:wrap;flex-wrap:wrap;z-index:2;justify-content:center;align-items:center;}#flexbox_2 > div{padding:0px;}#flexbox_back_3{background-color:rgba(44,63,84,0.5);width:100%;height:100%;padding-top:200px;padding-bottom:200px;min-height:99px;}#flexbox_3{display:-webkit-flex;;display:-moz-box;display:-ms-flexbox;display:flex;-webkit-flex-wrap:wrap;flex-wrap:wrap;z-index:3;justify-content:center;align-items:center;}#flexbox_3 > div{padding:0px;}#flexbox_back_4{background-color:rgba(254,255,255,1);width:100%;height:100%;padding-top:20px;padding-bottom:20px;min-height:112px;}#flexbox_4{display:-webkit-flex;;display:-moz-box;display:-ms-flexbox;display:flex;-webkit-flex-wrap:wrap;flex-wrap:wrap;z-index:4;justify-content:space-around;align-items:center;}#flexbox_4 > div{padding:0px;}#navmenu_menu0 {clear:both;float:left;margin:0;padding:0;width:100%;font-family:'WorkSans-Light','Work Sans','Helvetica';font-size:14px;z-index:1000;}#navmenu_menu0 ul {margin:0;padding:0;list-style:none;position:relative;right:0px;float:right;}#navmenu_menu0 ul li {margin:0 10px 0 0;padding:0;float:left;position:relative;left:0px;min-width: 15px;}#navmenu_menu0 ul li a {display:block;margin:0;padding:5px;height: 27px;line-height: 27px;text-decoration:none;color:#A9A9A8;font-weight: 300;text-align: left;}#navmenu_menu0 ul li a:visited {text-decoration:none;color:#A9A9A8;}#navmenu_menu0 ul li a:hover {text-decoration:none;color:#FE7D77;}#navmenu_menu0 ul li:hover a,#navmenu_menu0 ul li.hover a {text-decoration:none;color:#FE7D77;}#navmenu_menu0 ul ul {z-index:11006;display:none;position:absolute;left:0;float:left;right:auto;padding-top:2px;}#navmenu_menu0 ul ul li {left:auto;margin:0;clear:left;float:left;width:100%;}#navmenu_menu0 ul ul li a,#navmenu_menu0 ul li.active li a,#navmenu_menu0 ul li:hover ul li a,#navmenu_menu0 ul li.hover ul li a {background:#CBCBCB;text-decoration:none;color:#A9A9A8;padding-top:0px;padding-bottom:0px;border-bottom:1px solid #000000;float:left;width:5px;width:100%;text-align: left;white-space:nowrap;}#navmenu_menu0 ul ul li a:hover,#navmenu_menu0 ul li.active ul li a:hover,#navmenu_menu0 ul li:hover ul li a:hover,#navmenu_menu0 ul li.hover ul li a:hover {background:#CBCBCB;text-decoration:none;color:#FE7D77;float:left;}#navmenu_menu0 ul li.last ul {left:auto; right:0;}#navmenu_menu0 ul li.last ul li {float:right;position:relative;right:10px;}#navmenu_menu0 ul li:hover ul,#navmenu_menu0 ul li.hover ul {display:block;}</style></head><body><div class="container"><div class="content" data-minheight="1000"><div class="" id="flexbox_back_4"><div id="flexbox_4"><div><div class="shape_5" style="width:357px;min-height:112px;"><div style="padding: 42px 2.16px 0px 2.16px; "><p style="line-height:28px;text-align:center;margin-top:0px;margin-bottom:0px;" class="Style1">N<span class="Style8">ajmanovich</span><span class="Style9"> R</span><span class="Style8">esearch</span><span class="Style9"> G</span><span class="Style8">roup</span></p></div></div></div><div><div class="ewnavmenu" id="navmenu_menu0" style="left:858px;top:57px;height:37px;width:360px;z-index:10006;"><ul id="navigation_menu0"><li><a class="main item0" href="index.html" data-linkuuid="907768C0C4D34DAB898DCCF10CD97DEE">Home</a></li><li><a class="main item1" href="resources.html" data-linkuuid="907768C0C4D34DAB898DCCF10CD97DEE">Resources</a></li><li  style='margin-right:0px;' class="last"><a class="main item2" href="publications.html" data-linkuuid="907768C0C4D34DAB898DCCF10CD97DEE">Publications</a></li></ul></div></div></div></div><div class="" id="flexbox_back_3"><div id="flexbox_3"><div><div style="width:327px;animation-duration:1.0s;-webkit-animation-duration:1.0s;animation-delay:0.0s;-webkit-animation-delay:0.0s;animation-iteration-count:1;-webkit-animation-iteration-count:1;" data-fadeopt="fadeInUp" class="  animated"><!--327-->
	<div id="text_box_7" style="text-align:Center;min-width:300px"">
	<br />
	
	<span class="ql-font-helvetica" style="font-size: 12px;font-family:'Helvetica',sans-serif;"><h2><span class="ql-font-helvetica" style="color: rgb(255, 255, 255); font-size: 44px;font-family:'Helvetica',sans-serif;">PUBLICATIONS</span></h2></span>

	
	</div></div></div></div></div><div class="" id="flexbox_back_2"><div id="flexbox_2"><div><div class="shape_10" style="width:100%;min-height:2178px;"><div style="padding: 14.4px 14.4px 14.4px 14.4px; "><p style="line-height:18px;margin-top:0px;margin-bottom:11px;" class="Style10"><br /></p>

		<p style="line-height:21px;margin-bottom:11px;margin-top:0px;" class="Style11">61. Rodríguez-Salazar, C. A., Tol, S. van, Mailhot, O., Galdino, G. T., Teruel, N. F. B., Zhang, L., Warren, A. N., González-Orozco, M., Freiberg, A. N., Najmanovich, R. J., Giraldo, M. I. & Rajsbaum, R. (2023) Ebola Virus VP35 Interacts Non-Covalently with Ubiquitin Chains to Promote Viral Replication Creating New Therapeutic Opportunities. bioRxiv 2023.07.14.549057.</p>

		<p style="line-height:21px;margin-bottom:11px;margin-top:0px;" class="Style11">60. Teruel, Nátalia Fagundes Borges; Borges, Vinicius Magalhães; Najmanovich, Rafael (2023). Surfaces: A software to quantify and visualize interactions within and between proteins and ligands. bioRxiv 2023.04.26.538470.</p>

		<p style="line-height:21px;margin-bottom:11px;margin-top:0px;" class="Style11">59. Mailhot, Olivier; Major, François; Najmanovich, Rafael (2023) The DynaSig-ML Python package: automated learning of biomolecular dynamics–function relationships. Bioinformatics 39(4):btad180.</p>

		<p style="line-height:21px;margin-bottom:11px;margin-top:0px;" class="Style11">58. Pandey, Somnath; Djibo, Rahinatou; Darracq, Anaïs; Calendo, Gennaro; Zhang, Hanghang; Henry, Ryan A; Andrews, Andrew J; Baylin, Stephen B; Madzo, Jozef; Najmanovich, Rafael; (2022). Selective CDK9 Inhibition by Natural Compound Toyocamycin in Cancer Cells. Cancers 14(14): 3340.</p>

		<p style="line-height:21px;margin-bottom:11px;margin-top:0px;" class="Style11">57. Mailhot, Olivier; Frappier, Vincent; Major, François; Najmanovich, Rafael J (2022). Sequence-sensitive elastic network captures dynamical features necessary for miR-125a maturation. PLOS Computational Biology 18(12): e1010777.</p>

		<p style="line-height:21px;margin-bottom:11px;margin-top:0px;" class="Style11">56. Crown, Matthew; Teruel, Natalia; Najmanovich, Rafael; Bashton, Matthew (2022). SPEAR: Systematic ProtEin AnnotatoR. Bioinformatics 38(15): 3827-3829.</p>

		<p style="line-height:21px;margin-bottom:11px;margin-top:0px;" class="Style11">55. N Teruel, M Crown, M Bashton, R Najmanovich (2021). Computational analysis of the effect of SARS-CoV-2 variant Omicron Spike protein mutations on dynamics, ACE2 binding and propensity for immune escape. bioRxiv: 2021.12.14.472622.</p>
		
		<p style="line-height:21px;margin-bottom:11px;margin-top:0px;" class="Style11">54. Marielle Huot, Maxime Caron, Chantal Richer, Rahinatou Djibo, Rafael Najmanovich, Pascal St-Onge, Daniel Sinnett, Noël JM Raynal (2021). Repurposing proscillaridin A in combination with decitabine against embryonal rhabdomyosarcoma RD cells. Cancer chemotherapy and pharmacology 88 (5), 845-856<./p>

		<p style="line-height:21px;margin-bottom:11px;margin-top:0px;" class="Style11">53. O Mailhot, R Najmanovich (2021). The NRGTEN Python package: an extensible toolkit for coarse-grained normal mode analysis of proteins, nucleic acids, small molecules and their complexes. Bioinformatics 37 (19), 3369-3371.</p>

		<p style="line-height:21px;margin-bottom:11px;margin-top:0px;" class="Style11">52. N Teruel, O Mailhot, RJ Najmanovich (2021). Modelling conformational state dynamics and its role on infection for SARS-CoV-2 Spike protein variants. PLoS computational biology 17 (8), e1009286. </p>

		<p style="line-height:21px;margin-bottom:11px;margin-top:0px;" class="Style11">51. Bernier SC, Morency LP, Najmanovich R, Salesse C. Identification of an alternative translation initiation site in the sequence of the commonly used Glutathione S-Transferase tag. J Biotechnol. 2018 286:14-16. </p>

		<p style="line-height:21px;margin-bottom:11px;margin-top:0px;" class="Style11">50. Dion SP, BÈliveau F, Morency LP, DÈsilets A, Najmanovich R, Leduc R. Functional diversity of TMPRSS6 isoforms and variants expressed in hepatocellular carcinoma cell lines. Sci Rep. 2018 Aug 22;8(1):12562. </p>
		
		<p style="line-height:21px;margin-bottom:11px;margin-top:0px;" class="Style11">49. Briot J, Mailhot O, Bourdin B, TÈtreault MP, Najmanovich R, Parent L. A three-way inter-molecular network accounts for the CaVduced functional modulation of the pore-forming CaV1. 2 subunit. J Biol Chem. 2018 293(19):7176-7188. </p>
		
		<p style="line-height:21px;margin-bottom:11px;margin-top:0px;" class="Style11">48. Morency LP, Gaudreault F, Najmanovich R. Applications of the NRGsuite and the Molecular Docking Software FlexAID in Computational Drug Discovery and Design. Methods Mol Biol.1762:367-388.</p>
		
		<p style="line-height:21px;margin-bottom:11px;margin-top:0px;" class="Style11">47. Yan, LH, Le Roux A, Boyapelly K, Lamontagne AM, Archambault MA, Picard-Jean F, Lalonde-Seguin D, St-Pierre E, Najmanovich RJ, Fortier LC, Lafontaine D, Marsault E. Purine analogs targeting the guanine ribswitch as potential antibiotics against Clostridioides difficile. European Journal of Medicinal Chemistry. 2018; 143:755-768.</p>
		
		<p style="line-height:21px;margin-bottom:11px;margin-top:0px;" class="Style11">46. Najmanovich RJ. Evolutionary studies of ligand binding sites in proteins. Curr Opin Struct Biol. 2017;45: 85–90. </p>
		
		<p style="line-height:21px;margin-bottom:11px;margin-top:0px;" class="Style11">45. Chénard T, Guénard F, Vohl M-C, Carpentier A, Tchernof A, Najmanovich RJ. Remodeling adipose tissue through in silico modulation of fat storage for the prevention of type 2 diabetes. BMC Systems Biology. 2017;11: 615. </p>
		
		<p style="line-height:21px;margin-bottom:11px;margin-top:0px;" class="Style11">44. Chartier M, Morency L-P, Zylber MI, Najmanovich RJ. Large-scale detection of drug off-targets: hypotheses for drug repurposing and understanding side-effects. BMC Pharmacol Toxicol. 2017;18: 1046. </p>
		
		<p style="line-height:21px;margin-bottom:11px;margin-top:0px;" class="Style11">43. St-Georges C, Désilets A, Béliveau F, Ghinet M, Dion SP, Colombo E, Boudreault P-L, Najmanovich RJ, Leduc R & Marsault E. Modulating the selectivity of matriptase-2 inhibitors with unnatural amino acids. European Journal of Medicinal Chemistry. 2017;129: 110–123. </p>
		
		<p style="line-height:21px;margin-bottom:11px;margin-top:0px;" class="Style11">42. Frappier V, Chartier M, Najmanovich R. Applications of Normal Mode Analysis Methods in Computational Protein Design. Methods Mol Biol. 2017;1529: 203–214. </p>
		
		<p style="line-height:21px;margin-bottom:11px;margin-top:0px;" class="Style11">41. Belloli L, Wainer G, Najmanovich RJ. Parsing and model generation for biological processes. TMS-DEVS 16 Proceedings of the Symposium on Theory of Modelling & Simulation. 2016.</p>
		
		<p style="line-height:21px;margin-bottom:11px;margin-top:0px;" class="Style11">40. Gaudreault F, Morency L-P, Najmanovich RJ. NRGsuite: a PyMOL plugin to perform docking simulations in real time using FlexAID. Bioinformatics. 2015;31: 3856–3858. </p>
		
		<p style="line-height:21px;margin-bottom:11px;margin-top:0px;" class="Style11">39. Chartier M, Adriansen E, Najmanovich R. IsoMIF Finder: online detection of binding site molecular interaction field similarities. Bioinformatics. 2015;32: btv616–623. </p>
		
		<p style="line-height:21px;margin-bottom:11px;margin-top:0px;" class="Style11">38. Chartier M, Najmanovich R. Detection of Binding Site Molecular Interaction Field Similarities. J Chem Inf Model. 2015;55: 1600–1615.</p>
		
		<p style="line-height:21px;margin-bottom:11px;margin-top:0px;" class="Style11">37. Frappier V, Chartier M, Najmanovich RJ. ENCoM server: exploring protein conformational space and the effect of mutations on protein function and stability. Nucleic Acids Res. 2015;43: W395–400. </p>
		
		<p style="line-height:21px;margin-bottom:11px;margin-top:0px;" class="Style11">36. Gaudreault F, Najmanovich RJ. FlexAID: Revisiting Docking on Non-Native-Complex Structures. J Chem Inf Model. 2015;55: 1323–1336. </p>
		
		<p style="line-height:21px;margin-bottom:11px;margin-top:0px;" class="Style11">35. Frappier V, Najmanovich RJ. Vibrational entropy differences between mesophile and thermophile proteins and their use in protein engineering. Protein Science. 2015;24: 474–483. </p>
		
		<p style="line-height:21px;margin-bottom:11px;margin-top:0px;" class="Style11">34. Samish I, Bourne PE, Najmanovich RJ. Achievements and challenges in structural bioinformatics and computational biophysics. Bioinformatics. 2015;31: 146–150. </p>
		
		<p style="line-height:21px;margin-bottom:11px;margin-top:0px;" class="Style11">33. Duchêne D, Colombo E, Désilets A, Boudreault P-L, Leduc R, Marsault E, Najmanovich RJ. Analysis of subpocket selectivity and identification of potent selective inhibitors for matriptase and matriptase-2. J Med Chem. 2014;57: 10198–10204. </p>
		
		<p style="line-height:21px;margin-bottom:11px;margin-top:0px;" class="Style11">32. Larocque M, Chénard T, Najmanovich RJ. A curated C. difficile strain 630 metabolic network: prediction of essential targets and inhibitors. BMC Systems Biology. BioMed Central Ltd; 2014;8: 117.</p>
		
		<p style="line-height:21px;margin-bottom:11px;margin-top:0px;" class="Style11">31. Frappier V, Najmanovich RJ. A coarse-grained elastic network atom contact model and its use in the simulation of protein dynamics and the prediction of the effect of mutations. MacKerell AD, editor. PLoS Comput Biol. 2014;10: e1003569. </p>
		
		<p style="line-height:21px;margin-bottom:11px;margin-top:0px;" class="Style11">30. Glouzon J-PS, Bolduc F, Wang S, Najmanovich RJ, Perreault J-P. Deep-sequencing of the peach latent mosaic viroid reveals new aspects of population heterogeneity. PLoS ONE. 2014;9: e87297. </p>
		
		<p style="line-height:21px;margin-bottom:11px;margin-top:0px;" class="Style11">29. Kurbatova N, Chartier M, Zylber MI, Najmanovich RJ. IsoCleft Finder - a web-based tool for the detection and analysis of protein binding-site geometric and chemical similarities. F1000Res. 2013;2: 117. </p>
		
		<p style="line-height:21px;margin-bottom:11px;margin-top:0px;" class="Style11">28. Chartier M, Chénard T, Barker J, Najmanovich RJ. Kinome Render: a stand-alone and web-accessible tool to annotate the human protein kinome tree. PeerJ. PeerJ Inc; 2013;1: e126. </p>
		
		<p style="line-height:21px;margin-bottom:11px;margin-top:0px;" class="Style11">27. Rosado LA, Vasconcelos IB, Palma MS, Frappier V, Najmanovich RJ, Santos DS, Basso LA. The Mode of Action of Recombinant Mycobacterium tuberculosis Shikimate Kinase: Kinetics and Thermodynamics Analyses. PLoS ONE. 2013;8: e61918</p>
		
		<p style="line-height:21px;margin-bottom:11px;margin-top:0px;" class="Style11">26. Letourneau D, Lorin A, Lefebvre A, Frappier V, Gaudreault F, Najmanovich R, Lavigne P, Lehoux J-G. StAR-related lipid transfer domain protein 5 binds primary bile acids. J Lipid Res. 2012;53: 2677–2689.</p>
		
		<p style="line-height:21px;margin-bottom:11px;margin-top:0px;" class="Style11">25. Gaudreault F, Chartier M, Najmanovich RJ. Side-chain rotamer changes upon ligand binding: common, crucial, correlate with entropy and rearrange hydrogen bonding. Bioinformatics. 2012;28: i423–i430.</p>
		
		<p style="line-height:21px;margin-bottom:11px;margin-top:0px;" class="Style11">24. Colombo E, Désilets A, Duchêne D, Chagnon F, Najmanovich RJ, Leduc R, Marsault E. Design and synthesis of potent, selective inhibitors of matriptase. ACS Med Chem Lett. 2012;3: 530–534. </p>
		
		<p style="line-height:21px;margin-bottom:11px;margin-top:0px;" class="Style11">23. Chartier M, Gaudreault F, Najmanovich R. Large-scale analysis of conserved rare codon clusters suggests an involvement in co-translational molecular recognition events. Bioinformatics. 2012;28: 1438–1445.</p>
		
		<p style="line-height:21px;margin-bottom:11px;margin-top:0px;" class="Style11">22. Campagna-Slater V, Mok MW, Nguyen KT, Feher M, Najmanovich R, Schapira M. Structural chemistry of the histone methyltransferases cofactor binding site. J Chem Inf Model. 2011;51: 612–623.</p>
		
		<p style="line-height:21px;margin-bottom:11px;margin-top:0px;" class="Style11">21. Han GW, Bakolitsa C, Miller MD, Kumar A, Carlton D, Najmanovich RJ, Abdubek P, Astakhova T, Axelrod HL, Chen C, Chiu H-J, Clayton T, Das D, Deller MC, Duan L, Ernst D, Feuerhelm J, Grant JC, Grzechnik A, Jaroszewski L, Jin KK, Johnson HA, Klock HE, Knuth MW, Kozbial P, Krishna SS, Marciano D, McMullan D, Morse AT, Nigoghossian E, Okach L, Reyes R, Rife CL, Sfcovic N, Tien HJ, Trame CB, van de Bedem H, Weekes D, Xu Q, Hodgson KO, Wooley J, Elsliger M-A, Deacon AM, Godzik A, Lesley SA & Wilson IA. Structures of the first representatives of Pfam family PF06938 (DUF1285) reveal a new fold with repeated structural motifs and possible involvement in signal transduction. Acta Crystallogr Sect F Struct Biol Cryst Commun. 2010;66: 1218–1225. </p>
		
		<p style="line-height:21px;margin-bottom:11px;margin-top:0px;" class="Style11">20. Bakolitsa C, Kumar A, McMullan D, Krishna SS, Miller MD, Carlton D, Najmanovich RJ, Abdubek, P, Astakhova T, Chiu H-J, Clayton T, Deller MC, Duan L, Elias Y, Feuerhelm J, Grant JC, Grzechnik SK, Han GW, Jaroszewski L, Jin KK, Klock HE, Knuth MW, Kozbial P, Marciano D, Morse AT, Nigoghossian E, Okach L, Oommachen S, Paulsen J, Reyes R, Rife CL, Trout CV, van de Bedem H, Weekes D, White A, Xu Q, Hodgson KO, Wooley J, Elsliger M-A, Deacon AM, Godzik A, Lesley SA & Wilson IA. The structure of the first representative of Pfam family PF06475 reveals a new fold with possible involvement in glycolipid metabolism. Acta Crystallogr Sect F Struct Biol Cryst Commun. 2010;66: 1211–1217.</p>
		
		<p style="line-height:21px;margin-bottom:11px;margin-top:0px;" class="Style11">19. Lavigne P, Najmanovich RJ, Lehoux J-G. Mammalian StAR-related lipid transfer (START) domains with specificity for cholesterol: structural conservation and mechanism of reversible binding. Subcell Biochem. 2010;51: 425–437. </p>
		
		<p style="line-height:21px;margin-bottom:11px;margin-top:0px;" class="Style11">18. Herrera JL, Gonzalez-Rey E, Fernandez-Montesinos R, Quintana FJ, Najmanovich R, Pozo D. Toll-like receptor stimulation differentially regulates vasoactive intestinal peptide type 2 receptor in macrophages. J Cell Mol Med. 2009;13: 3209–3217. </p>
		
		<p style="line-height:21px;margin-bottom:11px;margin-top:0px;" class="Style11">17. Andreini C, Bertini I, Cavallaro G, Najmanovich RJ, Thornton JM. Structural analysis of metal sites in proteins: non-heme iron sites as a case study. J Mol Biol. 2009;388: 356–380.&nbsp;&nbsp;</p>
		
		<p style="line-height:21px;margin-bottom:11px;margin-top:0px;" class="Style11">16. Najmanovich R, Kurbatova N, Thornton J. Detection of 3D atomic similarities and their use in the discrimination of small molecule protein-binding sites. Bioinformatics. 2008;24: i105–11. </p>
		
		<p style="line-height:21px;margin-bottom:11px;margin-top:0px;" class="Style11">15. Eswaran J, Bernad A, Ligos JM, Guinea B, Debreczeni JE, Sobott F, Parker SA, Najmanovich RJ, Turk BE, Knapp S. Structure of the human protein kinase MPSK1 reveals an atypical activation loop architecture. Structure. 2008;16: 115–124. </p>
		
		<p style="line-height:21px;margin-bottom:11px;margin-top:0px;" class="Style11">14. Allali-Hassani A*, Pan PW*, Dombrovski L*, Najmanovich R*, Tempel W, Dong A, Loppnau P, Martin F, Thornton JM, Edwards AM, Bochkarev A, Plotnikov AN, Vedadi M, Arrowsmith CH. Structural and chemical profiling of the human cytosolic sulfotransferases. PLoS Biol. 2007;5: e97. (* Equal contributing authors) </p>
		
		<p style="line-height:21px;margin-bottom:11px;margin-top:0px;" class="Style11">13. Najmanovich RJ, Allali-Hassani A, Morris RJ, Dombrovsky L, Pan PW, Vedadi M, Plotnikov AN, Edwards A, Arrowsmith C & Thornton JM. Analysis of binding site similarity, small-molecule similarity and experimental binding profiles in the human cytosolic sulfotransferase family. Bioinformatics. 2007;23: e104–9.</p>
		
		<p style="line-height:21px;margin-bottom:11px;margin-top:0px;" class="Style11">12. Glaser F, Morris RJ, Najmanovich RJ, Laskowski RA, Thornton JM. A method for localizing ligand binding pockets in protein structures. Proteins. 2006;62: 479–488.</p>
		
		<p style="line-height:21px;margin-bottom:11px;margin-top:0px;" class="Style11">11. Najmanovich RJ, Torrance JW, Thornton JM. Prediction of protein function from structure: insights from methods for the detection of local structural similarities. BioTechniques. 2005;38: 847, 849, 851. </p>
		
		<p style="line-height:21px;margin-bottom:11px;margin-top:0px;" class="Style11">10. Morris RJ, Najmanovich RJ, Kahraman A, Thornton JM. Real spherical harmonic expansion coefficients as 3D shape descriptors for protein binding pocket and ligand comparisons. Bioinformatics. 2005;21: 2347–2355. </p>
		
		<p style="line-height:21px;margin-bottom:11px;margin-top:0px;" class="Style11">9. Funkhouser, T., Glaser, F., Laskowski, R., Morris, R., Najmanovich, R., Stockwell, G., & Thornton, J. Shape-based classification of bound ligands. In S Barber, PD Baxter, KVMardia, & RE Walls (Eds), Quantitative Biology, Shape Analysis, and Wavelets. 2005; 39–42. </p>
		
		<p style="line-height:21px;margin-bottom:11px;margin-top:0px;" class="Style11">8. Morris RJ, Kahraman A, Funkhauser T, Najmanovich RJ, Stockwell G, Glaser F, Laskowski, R., Binding pocket shape analysis for protein function prediction. In S Barber, PD Baxter, KVMardia, & RE Walls (Eds), Quantitative Biology, Shape Analysis, and Wavelets. 2005; 91–94.</p>
		
		<p style="line-height:21px;margin-bottom:11px;margin-top:0px;" class="Style11">7. Eyal E, Najmanovich R, McConkey BJ, Edelman M, Sobolev V. Importance of solvent accessibility and contact surfaces in modeling side-chain conformations in proteins. J Comput Chem. 2004;25: 712–724. doi:10.1002/jcc.10420.</p>
		
		<p style="line-height:21px;margin-bottom:11px;margin-top:0px;" class="Style11">6. Eyal E, Najmanovich R, Edelman M, Sobolev V. Protein side-chain rearrangement in regions of point mutations. Proteins. 2003;50: 272–282. doi:10.1002/prot.10276.</p>
		
		<p style="line-height:21px;margin-bottom:11px;margin-top:0px;" class="Style11">5. Eyal E, Najmanovich RJ, Sobolev V, Edelman M. MutaProt: a web interface for structural analysis of point mutations. Bioinformatics. 2001;17: 381–382.</p>
		
		<p style="line-height:21px;margin-bottom:11px;margin-top:0px;" class="Style11">4. Najmanovich RJ, Kuttner J, Sobolev V, Edelman M. Side-chain flexibility in proteins upon ligand binding. PROTEINS: Structure, Function and Genetics. 2000;39: 261–268. </p>
		
		<p style="line-height:21px;margin-bottom:11px;margin-top:0px;" class="Style11">3. Vendruscolo M, Najmanovich RJ, Domany E. Can a pairwise contact potential stabilize native protein folds against decoys obtained by threading? PROTEINS: Structure, Function and Genetics. 2000;38: 134–148.</p>
		
		<p style="line-height:21px;margin-bottom:11px;margin-top:0px;" class="Style11">2. Vendruscolo M, Najmanovich RJ, Domany E. Protein folding in contact map space. Phys Rev Lett. 1999;82: 656–659<span class="Style12">.</span></p>
		
		<p style="line-height:21px;margin-bottom:11px;margin-top:0px;" class="Style11">1. Najmanovich RJ, deLyra J, Henriques V. The collapse transition in the HP model. Physica A. 1998;249: 374–379.</p>
	
	</div></div></div></div></div></div><footer data-top='2829' data-height='644'><div class="" id="flexbox_back_1"><div id="flexbox_1"><div><div class="shape_8" style="width:428px;min-height:314px;"><div style="padding: 36px 36px 36px 36px; "><p style="line-height:25px;text-align:center;margin-top:0px;margin-bottom:11px;" class="Style2">Contact Infromation</p><p style="line-height:19px;margin-bottom:11px;margin-top:0px;" class="Style3">Department of Pharmacology & Physiology</p><p style="line-height:19px;margin-bottom:11px;margin-top:0px;" class="Style3">Faculty of Medicine</p><p style="line-height:19px;margin-bottom:11px;margin-top:0px;" class="Style3">Université de Montreal</p><p style="line-height:19px;margin-bottom:11px;margin-top:0px;" class="Style3"><br /></p><p style="line-height:19px;margin-bottom:11px;margin-top:0px;" class="Style3">Room 3147 - Pavillion Paul-G.-Desmarais</p><p style="line-height:19px;margin-bottom:11px;margin-top:0px;" class="Style3">2960 Chemin de la Tour - H3T 1J4</p><p style="line-height:19px;margin-bottom:11px;margin-top:0px;" class="Style3">Montreal, QC - Canada</p></div></div></div></div></div><div class="" id="flexbox_back_0"><div id="flexbox_0"><div><div class="shape_9" style="width:212px;min-height:100px;"><img src="images/publications/Logo_UdeM-blanc-RVB.png" height="100" width="212" /></div></div></div></div></footer></div></body></html>