<!DOCTYPE html><html><head><title></title><meta http-equiv="Content-type" content="text/html;charset=UTF-8" /><meta name="viewport" content="width=device-width, initial-scale=1"><meta name="description" content="" /><meta name="generator" content="EverWeb 2.9.1 (2184)" /><meta name="buildDate" content="Wednesday, February 27, 2019" /><meta property="og:url" content="http://biophys.umontreal.ca/nrg/resources.html" /><meta property="og:type" content="website" /> <link rel="stylesheet" type="text/css" href="ew_css/textstyles.css?3634139505" /><link rel="stylesheet" type="text/css" href="ew_css/responsive.css?3634139505" /><!--[if lt IE 9]>
<script src="//ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"></script>
<!--<![endif]-->
<!--[if (gte IE 9) | (!IE)]><!-->
<script src="//ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
<!--<![endif]-->
<script type="text/javascript">
if (typeof jQuery == 'undefined')
{
<!--[if lt IE 9]>
 document.write(unescape("%3Cscript src='ew_js/jquery-1.9.1.min' type='text/javascript'%3E%3C/script%3E"));
<!--<![endif]-->
<!--[if (gte IE 9) | (!IE)]><!-->
 document.write(unescape("%3Cscript src='ew_js/jquery.min.js' type='text/javascript'%3E%3C/script%3E"));
<!--<![endif]-->
}
</script><script type="text/javascript" src="ew_js/imageCode.js"></script><script type="text/javascript" src="ew_js/viewchecker.js"></script><link rel="stylesheet" type="text/css" href="ew_css/animate.css" /><link rel="stylesheet" type="text/css" href="//fonts.googleapis.com/css?family=Work+Sans:400,700,300"><style>
	div#text_box_7 .tb_button { font-family:'ArialMT','Arial';font-size:16px;color:#D5D5D5;text-decoration:none; }
	
</style><style>div#text_box_7 a {color: rgb(4,50,255);text-decoration: underline;}div#text_box_7 a:hover {color: rgb(255,38,0);text-decoration: underline;}div#text_box_7 a:visited {color: rgb(148,32,146);text-decoration: underline;}</style><script type="text/javascript">jQuery(document).ready(function(){jQuery(".animated").addClass("animhidden").viewportChecker({callbackFunction:function(a){var b=a.data("fadeopt");a.hasClass(b)||a.addClass(b)},classToAdd:"animvisible",classToRemove:"animhidden",offset:100})});</script><style>
	div#text_box_10 .tb_button { font-family:'ArialMT','Arial';font-size:16px;color:#D5D5D5;text-decoration:none; }
	div#text_box_10 img#pltsTop { margin: 20px;}
</style><style>div#text_box_10 a {color: rgb(4,50,255);text-decoration: underline;}div#text_box_10 a:hover {color: rgb(255,38,0);text-decoration: underline;}div#text_box_10 a:visited {color: rgb(148,32,146);text-decoration: underline;}</style><style>
	div#text_box_11 .tb_button { font-family:'ArialMT','Arial';font-size:16px;color:#D5D5D5;text-decoration:none; }
	div#text_box_11 img#pltsTop { margin: 20px;}
</style><style>div#text_box_11 a {color: rgb(4,50,255);text-decoration: underline;}div#text_box_11 a:hover {color: rgb(255,38,0);text-decoration: underline;}div#text_box_11 a:visited {color: rgb(148,32,146);text-decoration: underline;}</style><style>
	div#text_box_12 .tb_button { font-family:'ArialMT','Arial';font-size:16px;color:#D5D5D5;text-decoration:none; }
	div#text_box_12 img#pltsTop { margin: 20px;}
</style><style>div#text_box_12 a {color: rgb(4,50,255);text-decoration: underline;}div#text_box_12 a:hover {color: rgb(255,38,0);text-decoration: underline;}div#text_box_12 a:visited {color: rgb(148,32,146);text-decoration: underline;}</style><style>
	div#text_box_13 .tb_button { font-family:'ArialMT','Arial';font-size:16px;color:#D5D5D5;text-decoration:none; }
	div#text_box_13 img#pltsTop { margin: 20px;}
</style><style>div#text_box_13 a {color: rgb(4,50,255);text-decoration: underline;}div#text_box_13 a:hover {color: rgb(255,38,0);text-decoration: underline;}div#text_box_13 a:visited {color: rgb(148,32,146);text-decoration: underline;}</style><style>
	div#text_box_14 .tb_button { font-family:'ArialMT','Arial';font-size:16px;color:#D5D5D5;text-decoration:none; }
	div#text_box_14 img#pltsTop { margin: 20px;}
</style><style>div#text_box_14 a {color: rgb(4,50,255);text-decoration: underline;}div#text_box_14 a:hover {color: rgb(255,38,0);text-decoration: underline;}div#text_box_14 a:visited {color: rgb(148,32,146);text-decoration: underline;}</style><style>
	div#text_box_15 .tb_button { font-family:'ArialMT','Arial';font-size:16px;color:#D5D5D5;text-decoration:none; }
	div#text_box_15 img#pltsTop { margin: 8px;}
</style><style>div#text_box_15 a {color: rgb(4,50,255);text-decoration: underline;}div#text_box_15 a:hover {color: rgb(255,38,0);text-decoration: underline;}div#text_box_15 a:visited {color: rgb(148,32,146);text-decoration: underline;}</style><style type="text/css">a img {border:0px;}div.content {min-height:1000px;}footer  {min-height:100px;}body {background: url('images/resources/browser/1stp.png') no-repeat center center fixed;-webkit-background-size: cover;-moz-background-size: cover;-o-background-size: cover;background-size: cover;background-color: #FEFFFF;margin: 0px 0 0 0;}div.container {width: 100%;}@media all and (max-device-width: 768px),(max-device-width: 568px),(max-device-width: 480px),(max-device-width: 667px),(max-device-width: 736px),(max-device-width: 360px)   {body {background: url('images/resources/browser/1stp-768px.png') no-repeat center center fixed;-webkit-background-size: cover;-moz-background-size: cover;-o-background-size: cover;background-size: cover;background-color: #FEFFFF;}}#flexbox_back_0{background-color:rgba(0,0,0,0.8);width:100%;height:100%;padding-top:25px;padding-bottom:25px;min-height:130px;}#flexbox_0{display:-webkit-flex;;display:-moz-box;display:-ms-flexbox;display:flex;-webkit-flex-wrap:wrap;flex-wrap:wrap;z-index:0;justify-content:center;align-items:center;}#flexbox_0 > div{padding:15px;}#flexbox_back_1{background-color:rgba(145,145,145,0.9);width:100%;height:100%;padding-top:75px;padding-bottom:75px;min-height:314px;}#flexbox_1{display:-webkit-flex;;display:-moz-box;display:-ms-flexbox;display:flex;-webkit-flex-wrap:wrap;flex-wrap:wrap;z-index:1;justify-content:center;align-items:flex-start;}#flexbox_1 > div{padding:0px;}#flexbox_back_2{background-color:rgba(254,255,255,1);width:100%;height:100%;padding-top:0px;padding-bottom:0px;min-height:823px;}#flexbox_2{display:-webkit-flex;;display:-moz-box;display:-ms-flexbox;display:flex;-webkit-flex-wrap:wrap;flex-wrap:wrap;z-index:2;justify-content:center;align-items:center;}#flexbox_2 > div{padding:10px;}#flexbox_back_3{background-color:rgba(44,63,84,0.5);width:100%;height:100%;padding-top:200px;padding-bottom:200px;min-height:99px;}#flexbox_3{display:-webkit-flex;;display:-moz-box;display:-ms-flexbox;display:flex;-webkit-flex-wrap:wrap;flex-wrap:wrap;z-index:3;justify-content:center;align-items:center;}#flexbox_3 > div{padding:0px;}#flexbox_back_4{background-color:rgba(254,255,255,1);width:100%;height:100%;padding-top:20px;padding-bottom:20px;min-height:112px;}#flexbox_4{display:-webkit-flex;;display:-moz-box;display:-ms-flexbox;display:flex;-webkit-flex-wrap:wrap;flex-wrap:wrap;z-index:4;justify-content:space-around;align-items:center;}#flexbox_4 > div{padding:0px;}#navmenu_menu0 {clear:both;float:left;margin:0;padding:0;width:100%;font-family:'WorkSans-Light','Work Sans','Helvetica';font-size:14px;z-index:1000;}#navmenu_menu0 ul {margin:0;padding:0;list-style:none;position:relative;right:0px;float:right;}#navmenu_menu0 ul li {margin:0 10px 0 0;padding:0;float:left;position:relative;left:0px;min-width: 15px;}#navmenu_menu0 ul li a {display:block;margin:0;padding:5px;height: 27px;line-height: 27px;text-decoration:none;color:#A9A9A8;font-weight: 300;text-align: left;}#navmenu_menu0 ul li a:visited {text-decoration:none;color:#A9A9A8;}#navmenu_menu0 ul li a:hover {text-decoration:none;color:#FE7D77;}#navmenu_menu0 ul li:hover a,#navmenu_menu0 ul li.hover a {text-decoration:none;color:#FE7D77;}#navmenu_menu0 ul ul {z-index:11006;display:none;position:absolute;left:0;float:left;right:auto;padding-top:2px;}#navmenu_menu0 ul ul li {left:auto;margin:0;clear:left;float:left;width:100%;}#navmenu_menu0 ul ul li a,#navmenu_menu0 ul li.active li a,#navmenu_menu0 ul li:hover ul li a,#navmenu_menu0 ul li.hover ul li a {background:#CBCBCB;text-decoration:none;color:#A9A9A8;padding-top:0px;padding-bottom:0px;border-bottom:1px solid #000000;float:left;width:5px;width:100%;text-align: left;white-space:nowrap;}#navmenu_menu0 ul ul li a:hover,#navmenu_menu0 ul li.active ul li a:hover,#navmenu_menu0 ul li:hover ul li a:hover,#navmenu_menu0 ul li.hover ul li a:hover {background:#CBCBCB;text-decoration:none;color:#FE7D77;float:left;}#navmenu_menu0 ul li.last ul {left:auto; right:0;}#navmenu_menu0 ul li.last ul li {float:right;position:relative;right:10px;}#navmenu_menu0 ul li:hover ul,#navmenu_menu0 ul li.hover ul {display:block;}</style></head><body><div class="container"><div class="content" data-minheight="1000"><div class="" id="flexbox_back_4"><div id="flexbox_4"><div><div class="shape_5" style="width:357px;min-height:112px;"><div style="padding: 42px 2.16px 0px 2.16px; "><p style="line-height:28px;text-align:center;margin-top:0px;margin-bottom:0px;" class="Style1">N<span class="Style8">ajmanovich</span><span class="Style9"> R</span><span class="Style8">esearch</span><span class="Style9"> G</span><span class="Style8">roup</span></p></div></div></div><div><div class="ewnavmenu" id="navmenu_menu0" style="left:858px;top:57px;height:37px;width:360px;z-index:10006;"><ul id="navigation_menu0"><li><a class="main item0" href="index.html" data-linkuuid="19912EAC80EA4D629CCB470186CA0FB8">Home</a></li><li><a class="main item1" href="resources.html" data-linkuuid="19912EAC80EA4D629CCB470186CA0FB8">Resources</a></li><li  style='margin-right:0px;' class="last"><a class="main item2" href="publications.html" data-linkuuid="19912EAC80EA4D629CCB470186CA0FB8">Publications</a></li></ul></div></div></div></div><div class="" id="flexbox_back_3"><div id="flexbox_3"><div><div style="width:327px;animation-duration:1.0s;-webkit-animation-duration:1.0s;animation-delay:0.0s;-webkit-animation-delay:0.0s;animation-iteration-count:1;-webkit-animation-iteration-count:1;" data-fadeopt="fadeInUp" class="  animated"><!--327-->
	<div id="text_box_7" style="text-align:Center;min-width:300px"">
	<br />
	
	<span class="ql-font-helvetica" style="font-size: 12px;font-family:'Helvetica',sans-serif;"><h2><span class="ql-font-helvetica" style="font-size: 44px; color: rgb(255, 255, 255);font-family:'Helvetica',sans-serif;">RESOURCES</span></h2></span>

	
	</div></div></div></div></div><div class="" id="flexbox_back_2"><div id="flexbox_2"><div><div style="width:346px;" class=""><!--346-->
	<div id="text_box_10" style="text-align:Center;min-width:300px"">
	
	<img id="pltsTop" style="object-fit: scale-down;" src="ewExternalFiles/Figure_S1.png" width="300" height="auto"/>
	<br />
	
	<span class="ql-font-helvetica" style="font-size: 12px;font-family:'Helvetica',sans-serif;"><h2 class="ql-align-center" style="text-align:center"><span style="font-size: 44px;">FlexAID</span></h2><p><br></p><p class="ql-align-justify" style="text-align:justify">Small-molecule protein docking is an essential tool in drug design and to understand molecular recognition.&nbsp;Our group is responsible for the development of FlexAID [36].&nbsp;FlexAID, is a docking algorithm that can use small-molecules and peptides as ligands and proteins/nucleic acids as targets. It permits full ligand flexibility as well as target side-chain flexibility. FlexAID utilizes a soft scoring function, i.e. one that is not highly dependent on specific geometric criteria, based on surface complementarity. The energy parameters of the scoring function were derived from the classification of a largedataset of native and near native (less than 2Å&nbsp;RMSD) conformations for nearly 1500 complexes from the PDBbind database as true positive examples. These were countered over successive rounds of Monte Carlo optimization over an ever increasing and successively more difficult sets (increasingly lower energy decoys) with RMSD above 2Å. FlexAID has been used in numerous projects in collaborations and in our group.&nbsp;</p><p class="ql-align-justify" style="text-align:justify">FlexAID has been shown to outperform existing widely used software such as AutoDock Vina and FlexX in the prediction of binding poses. This is particularly true in cases where target flexibility is crucial, such as is likely to be the case when using homology models. The FlexAID source code is available at our GitHub page.</p></span>

	
	<div style="display: flex;flex-direction:row;flex-wrap: wrap;justify-content: center;align-items:center;">
	<div style="flex-grow:2;flex-shrink:2;margin:15px;min-width:150px;max-width:250px;"><a target="_blank" style="text-decoration:none;" href="https://github.com/NRGlab/FlexAID"><div class="tb_button" style='background-color: #FF0000;padding: 15px 10px 15px 10px;border-radius: 50px;text-decoration:none;text-align:center;color: #FFFFFF;' onmouseover="this.style.background='#FB0106';this.style.color='#FFFFFF';this.style.border='0px solid #000000';" onmouseout="this.style.background='#FF0000';this.style.color='#FFFFFF';this.style.border='0px solid #000000';">Get Source Code</div></a></div>
	</div>
	
	</div></div></div><div><div style="width:346px;" class=""><!--346-->
	<div id="text_box_11" style="text-align:Center;min-width:300px"">
	
	<img id="pltsTop" style="object-fit: scale-down;" src="ewExternalFiles/NRGsuite_fig.jpg" width="300" height="auto"/>
	<br />
	
	<span class="ql-font-helvetica" style="font-size: 12px;font-family:'Helvetica',sans-serif;"><h2 class="ql-align-center" style="text-align:center"><span style="font-size: 44px;">NRGsuite</span></h2><p class="ql-align-justify" style="text-align:justify">The NRGsuite is a PyMOL plugin that permits the detection of surface cavities in macromolecules using GetCleft, their refinement, calculation of volume and use, individually or jointly, as target binding-sites for docking simulations with FlexAID. The NRGsuite offers users control over a large number of important parameters in docking simulations including the assignment of flexile side-chains and the definition of geometric constraints (to simulate covalent docking). Furthermore, the NRGsuite permits the visualization of the docking simulation in real time.&nbsp;</p><p class="ql-align-justify" style="text-align:justify">The NRGsuite offers a convenient and user-friendly manner to perform docking simulations for structure-based drug design and the exploration of FlexAID simulation parameters as well as preparation of input files for large-scale runs of FlexAID on its own (for virtual screening). The NRGsuite is also an excellent tool for teaching. Please take the time to cite FlexAID&nbsp;and the NRGsuite [40]&nbsp;if you use it. You can download the <a href="ewExternalFiles/NRGsuite_Guide.pdf">user manual</a> to get started and the executables with the buttons below.</p></span>

	
	<div style="display: flex;flex-direction:row;flex-wrap: wrap;justify-content: center;align-items:center;">
	<div style="flex-grow:2;flex-shrink:2;margin:10px;min-width:70px;max-width:250px;"><a target="_blank" style="text-decoration:none;" href="ewExternalFiles/NRGsuite_2.48l_Linux32.tar.gz"><div class="tb_button" style='background-color: #FF0000;padding: 15px 10px 15px 10px;border-radius: 50px;text-decoration:none;text-align:center;color: #FFFFFF;' onmouseover="this.style.background='#FB0106';this.style.color='#FFFFFF';this.style.border='0px solid #000000';" onmouseout="this.style.background='#FF0000';this.style.color='#FFFFFF';this.style.border='0px solid #000000';">Linux 32-bit</div></a></div><div style="flex-grow:2;flex-shrink:2;margin:10px;min-width:70px;max-width:250px;"><a target="_blank" style="text-decoration:none;" href="ewExternalFiles/NRGsuite_2.48l_Linux64.tar.gz"><div class="tb_button" style='background-color: #FF0000;padding: 15px 10px 15px 10px;border-radius: 50px;text-decoration:none;text-align:center;color: #FFFFFF;' onmouseover="this.style.background='#FB0106';this.style.color='#FFFFFF';this.style.border='0px solid #000000';" onmouseout="this.style.background='#FF0000';this.style.color='#FFFFFF';this.style.border='0px solid #000000';">Linux 64-bit</div></a></div><div style="flex-grow:2;flex-shrink:2;margin:10px;min-width:70px;max-width:250px;"><a target="_blank" style="text-decoration:none;" href="ewExternalFiles/NRGsuite_2.48l_Win32.exe"><div class="tb_button" style='background-color: #FF0000;padding: 15px 10px 15px 10px;border-radius: 50px;text-decoration:none;text-align:center;color: #FFFFFF;' onmouseover="this.style.background='#FB0106';this.style.color='#FFFFFF';this.style.border='0px solid #000000';" onmouseout="this.style.background='#FF0000';this.style.color='#FFFFFF';this.style.border='0px solid #000000';">Windows 32-bit</div></a></div><div style="flex-grow:2;flex-shrink:2;margin:10px;min-width:70px;max-width:250px;"><a target="_blank" style="text-decoration:none;" href="ewExternalFiles/NRGsuite_2.48l_Win64.exe"><div class="tb_button" style='background-color: #FF0000;padding: 15px 10px 15px 10px;border-radius: 50px;text-decoration:none;text-align:center;color: #FFFFFF;' onmouseover="this.style.background='#FB0106';this.style.color='#FFFFFF';this.style.border='0px solid #000000';" onmouseout="this.style.background='#FF0000';this.style.color='#FFFFFF';this.style.border='0px solid #000000';">Windows 64-bit</div></a></div><div style="flex-grow:2;flex-shrink:2;margin:15px;min-width:150px;max-width:250px;"><a target="_blank" style="text-decoration:none;" href="ewExternalFiles/NRGsuite_2.48l_MacOSX64.pkg.zip"><div class="tb_button" style='background-color: #FF0000;padding: 15px 10px 15px 10px;border-radius: 50px;text-decoration:none;text-align:center;color: #FFFFFF;' onmouseover="this.style.background='#FB0106';this.style.color='#FFFFFF';this.style.border='0px solid #000000';" onmouseout="this.style.background='#FF0000';this.style.color='#FFFFFF';this.style.border='0px solid #000000';">MacOS 64-bit</div></a></div>
	</div>
	
	</div></div></div><div><div style="width:357px;" class=""><!--357-->
	<div id="text_box_12" style="text-align:Center;min-width:300px"">
	
	<img id="pltsTop" style="object-fit: scale-down;" src="ewExternalFiles/GetCleft_fig.jpg" width="300" height="auto"/>
	<br />
	
	<span class="ql-font-helvetica" style="font-size: 12px;font-family:'Helvetica',sans-serif;"><h2 class="ql-align-center" style="text-align:center"><span style="font-size: 44px;">GetCleft</span></h2><p><br></p><p class="ql-align-justify" style="text-align:justify">The detection of cavities, both internal or surface-exposed is very useful. Our GetCleft software is an implementation in C of the SurfNet algorithm developed by Laskowski [Laskowski, R. Surfnet - a Program for Visualizing Molecular-Surfaces, Cavities, and Intermolecular Interactions. 1995;13: 323–330]. We provide it here separately as it is a necessary independent step in the use of IsoMIF or FlexAID. However, if you use the NRGsuite, GetCleft is built-in and also gives user the possiblity to refine and measure the volume of cavities as shown in the image above. The GetCleft source code is available at our GitHub page.</p><p class="ql-align-justify" style="text-align:justify"><br></p><p class="ql-align-justify" style="text-align:justify"><br></p><p class="ql-align-justify" style="text-align:justify"><br></p></span>

	
	<div style="display: flex;flex-direction:row;flex-wrap: wrap;justify-content: center;align-items:center;">
	<div style="flex-grow:2;flex-shrink:2;margin:15px;min-width:150px;max-width:250px;"><a target="_blank" style="text-decoration:none;" href="https://github.com/NRGlab/Get_Cleft"><div class="tb_button" style='background-color: #FF0000;padding: 15px 10px 15px 10px;border-radius: 50px;text-decoration:none;text-align:center;color: #FFFFFF;' onmouseover="this.style.background='#FB0106';this.style.color='#FFFFFF';this.style.border='0px solid #000000';" onmouseout="this.style.background='#FF0000';this.style.color='#FFFFFF';this.style.border='0px solid #000000';">Get Source Code</div></a></div>
	</div>
	
	</div></div></div><div><div style="width:340px;" class=""><!--340-->
	<div id="text_box_13" style="text-align:Center;min-width:300px"">
	
	<img id="pltsTop" style="object-fit: scale-down;" src="ewExternalFiles/image8.png" width="300" height="auto"/>
	<br />
	
	<span class="ql-font-helvetica" style="font-size: 12px;font-family:'Helvetica',sans-serif;"><h2 class="ql-align-center" style="text-align:center"><span style="font-size: 44px;">ENCoM</span></h2><p class="ql-align-justify" style="text-align:justify"><br></p><p class="ql-align-justify" style="text-align:justify"><span style="color: black; background-color: rgb(255, 255, 255);font-family:'Helvetica',sans-serif;" class="ql-font-helvetica">ENCoM is a coarse-grained normal mode analysis method that is unique in the sense that the elastic constants the define the modes of movements are modulated by a potential force field and the surface area in contact between atoms belonging to different residues in the protein or nucleotides in RNA. The method has been applied to a number of questions, primarily thus far to proteins. We have used ENCoM to predict the effect of mutations on stability and applied it as a tool in protein engineering. We also use it to generate conformational ensembles that may be useful for a number of applications, including in the representation of ensembles for virtual screening. We are currently applying ENCoM to study allostery as well as signal transduction in GPCRs.&nbsp;</span><span style="color: black;font-family:'Helvetica',sans-serif;" class="ql-font-helvetica"> Whereas we have developed a very easy to use and convenient web-service to predict the effect of mutations and generate conformational ensembles, this is currently temporarily unavailable. All the results that could be obtained through the web interface can also be obtained via the stand-alone version of ENCoM that can be downloaded&nbsp;from&nbsp;GitHub (includes the necessary scripts). In order to facilitate its use, we have created a step-by-step tutorial. If you use ENCoM, please cite the original article [31] as well as the tutorial article [42</span>] f<span style="color: black;font-family:'Helvetica',sans-serif;" class="ql-font-helvetica">or which you will the scripts </span><a href="ewExternalFiles/ENCoM_script.zip" class="ql-font-helvetica" style="color: black;font-family:'Helvetica',sans-serif;">here</a><span style="color: black;font-family:'Helvetica',sans-serif;" class="ql-font-helvetica">.</span></p></span>

	
	<div style="display: flex;flex-direction:row;flex-wrap: wrap;justify-content: center;align-items:center;">
	<div style="flex-grow:2;flex-shrink:2;margin:15px;min-width:150px;max-width:250px;"><a target="_blank" style="text-decoration:none;" href="https://github.com/NRGlab/ENCoM"><div class="tb_button" style='background-color: #FF0000;padding: 15px 10px 15px 10px;border-radius: 50px;text-decoration:none;text-align:center;color: #FFFFFF;' onmouseover="this.style.background='#FB0106';this.style.color='#FFFFFF';this.style.border='0px solid #000000';" onmouseout="this.style.background='#FF0000';this.style.color='#FFFFFF';this.style.border='0px solid #000000';">Get Source Code</div></a></div>
	</div>
	
	</div></div></div><div><div style="width:353px;" class=""><!--353-->
	<div id="text_box_14" style="text-align:Center;min-width:300px"">
	
	<img id="pltsTop" style="object-fit: scale-down;" src="ewExternalFiles/IsoMif_example_Figure7.jpg" width="300" height="auto"/>
	<br />
	
	<span class="ql-font-helvetica" style="font-size: 12px;font-family:'Helvetica',sans-serif;"><h2 class="ql-align-center" style="text-align:center"><span style="font-size: 44px;">IsoMIF</span></h2><p><br></p><p class="ql-align-justify" style="text-align:justify"><span class="ql-font-helvetica" style="background-color: rgb(255, 255, 255); color: black;font-family:'Helvetica',sans-serif;">IsoMIF identifies binding site molecular interaction field similarities between proteins. IsoMIF allows you to identify binding site molecular interaction field (MIF) similarities between pairs of structures. You can also run your query structure against a database of pre-calculated MIFs or all-against-all comparisons of you own custom PDB entries.&nbsp;</span>As an example of IsoMIF, the figure shows the superimposition of the 3GEY and 1AHB (panel A) based on the MIF similarities found for 5 probes. The H-bond acceptor probes matched (Pannel B) shows the H-bonds made by 3GEY with the backbone amide of glycine 79 could be made by arginine 163 of 1AHB. While it doesn’t seem to be used in binding Pj34, the donor hydroxyl of serine 112 seems to have a counterpart with the backbone amide of isoleucine 71. Similarly there are many H-bond donor probes found and Figure (panel C) shows the acceptors atoms in each structure that are responsible for these similarities. Shared aromatic interactions are the result of favourable interaction energies between aromatic probes and tyrosine 117 or tyrosine 70 respectively that can make face-to-face aromatic interactions with the inhibitor (panel D). We deeply regret that the IsoMIF Server is not currently available due to technical problems. In the meantime, you can download the software to run locally and also the datasets of pre-calculated MIFs to run your query against. The,source code and scripts to visualize MIFs as well as matched similarities (release 150311) can be downloaded from GitHub. You can find <a href="ewExternalFiles/README.txt" target="_blank">here</a> the Instructions to compile and run IsoMIF locally. The following datasets used in the article above to compare IsoMIF to other methods can be downloaded <a href="ewExternalFiles/IsoMIF_Datasets.zip">here</a> containing: 1. Entries in the Kahraman, Homogenous, Steroid, Soippa, PDBbind and scPDB datasets (PDB, residue name, residue number, chain, alternate location); 2. True positive pairs for scPDB dataset; and 3. True positive pairs for PDBbind dataset.</p></span>

	
	<div style="display: flex;flex-direction:row;flex-wrap: wrap;justify-content: center;align-items:center;">
	<div style="flex-grow:2;flex-shrink:2;margin:15px;min-width:150px;max-width:250px;"><a target="_blank" style="text-decoration:none;" href="https://github.com/NRGlab/IsoMif"><div class="tb_button" style='background-color: #FF0000;padding: 15px 10px 15px 10px;border-radius: 50px;text-decoration:none;text-align:center;color: #FFFFFF;' onmouseover="this.style.background='#FB0106';this.style.color='#FFFFFF';this.style.border='0px solid #000000';" onmouseout="this.style.background='#FF0000';this.style.color='#FFFFFF';this.style.border='0px solid #000000';">Get Source Code</div></a></div>
	</div>
	
	</div></div></div><div><div style="width:358px;" class=""><!--358-->
	<div id="text_box_15" style="text-align:Center;min-width:300px"">
	
	<img id="pltsTop" style="object-fit: scale-down;" src="ewExternalFiles/Kinome_Rende_fig.png" width="300" height="auto"/>
	<br />
	
	<span class="ql-font-helvetica" style="font-size: 12px;font-family:'Helvetica',sans-serif;"><h2 class="ql-align-center" style="text-align:center"><span style="font-size: 44px;">Kinome Render</span></h2><p class="ql-align-justify" style="text-align:justify"><span style="font-size: 14px;">Human protein kinases play fundamental roles mediating the majority of signal transduction pathways in eukaryotic cells as well as a multitude of other processes involved in metabolism, cell-cycle regulation, cellular shape, motility, differentiation and apoptosis. The human protein kinome contains 518 members. Most studies that focus on the human kinome require, at some point, the visualization of large amounts of data. The visualization of such data within the framework of a phylogenetic tree may help identify key relationships between different protein kinases in view of their evolutionary distance and the information used to annotate the kinome tree. For example, studies that focus on the promiscuity of kinase inhibitors can benefit from the annotations to depict binding affinities across kinase groups. Images involving the mapping of information into the kinome tree are common. However, producing such figures manually can be a long arduous process prone to errors. To circumvent this issue, we have developed a stand-alone tool called Kinome Render (KR) that produces customized annotations on the human kinome tree. KR allows the creation and automatic overlay of customizable text or shape-based annotations of different sizes and colors on the human kinome tree.</span></p></span>

	
	<div style="display: flex;flex-direction:row;flex-wrap: wrap;justify-content: center;align-items:center;">
	<div style="flex-grow:2;flex-shrink:2;margin:15px;min-width:75px;max-width:250px;"><a target="_blank" style="text-decoration:none;" href="ewExternalFiles/KinomeRender_1.3Mac-Linux.zip"><div class="tb_button" style='background-color: #FF0000;padding: 15px 10px 15px 10px;border-radius: 50px;text-decoration:none;text-align:center;color: #FFFFFF;' onmouseover="this.style.background='#FB0106';this.style.color='#FFFFFF';this.style.border='0px solid #000000';" onmouseout="this.style.background='#FF0000';this.style.color='#FFFFFF';this.style.border='0px solid #000000';">MacOS and Linux</div></a></div><div style="flex-grow:2;flex-shrink:2;margin:15px;min-width:75px;max-width:250px;"><a target="_blank" style="text-decoration:none;" href="ewExternalFiles/KinomeRender_1.3Windows.zip"><div class="tb_button" style='background-color: #FF0000;padding: 15px 10px 15px 10px;border-radius: 50px;text-decoration:none;text-align:center;color: #FFFFFF;' onmouseover="this.style.background='#FB0106';this.style.color='#FFFFFF';this.style.border='0px solid #000000';" onmouseout="this.style.background='#FF0000';this.style.color='#FFFFFF';this.style.border='0px solid #000000';">Windows</div></a></div>
	</div>
	
	</div></div></div></div></div></div><footer data-top='2410' data-height='644'><div class="" id="flexbox_back_1"><div id="flexbox_1"><div><div class="shape_8" style="width:428px;min-height:314px;"><div style="padding: 36px 36px 36px 36px; "><p style="line-height:25px;text-align:center;margin-top:0px;margin-bottom:11px;" class="Style2">Contact Infromation</p><p style="line-height:19px;margin-bottom:11px;margin-top:0px;" class="Style3">Department of Pharmacology & Physiology</p><p style="line-height:19px;margin-bottom:11px;margin-top:0px;" class="Style3">Faculty of Medicine</p><p style="line-height:19px;margin-bottom:11px;margin-top:0px;" class="Style3">Université de Montreal</p><p style="line-height:19px;margin-bottom:11px;margin-top:0px;" class="Style3"><br /></p><p style="line-height:19px;margin-bottom:11px;margin-top:0px;" class="Style3">Room 3147 - Pavillion Paul-G.-Desmarais</p><p style="line-height:19px;margin-bottom:11px;margin-top:0px;" class="Style3">2960 Chemin de la Tour - H3T 1J4</p><p style="line-height:19px;margin-bottom:11px;margin-top:0px;" class="Style3">Montreal, QC - Canada</p></div></div></div></div></div><div class="" id="flexbox_back_0"><div id="flexbox_0"><div><div class="shape_9" style="width:212px;min-height:100px;"><img src="images/resources/Logo_UdeM-blanc-RVB.png" height="100" width="212" /></div></div></div></div></footer></div></body></html>